HelpSource/Classes/Splay.schelp

```diff
@@ -70,12 +83,36 @@ x = { |spread = 1, level = 0.2, center = 0.0|
 }.play;
 )
 
+::
+NOTE:: levelComp new takes multiple input options:
+true, false as before for rough equal power level compensation,
+and newly added: a float between 0 and 1. This is to offer more choices for level compensation, such as equal maximum amplitude.
+::
 
-// with mouse control
-(
-x = { var src;
-	src = SinOsc.ar({ |i| LFNoise2.kr(rrand(10, 20), 200, i + 3 * 100) } ! 10);
-	Splay.ar(src, MouseY.kr(1, 0), 0.2, MouseX.kr(-1, 1));
-}.play;
-)
+For a full discussion, see link::Classes/LevelComp::;
+here is an overview of the levelComp variants:
+
+code::
+// default is equal power comp: level / ( numchans.sqrt)
+Splay.ar(ins, spread, level, center ); // true is default
+// write it explicitly for clarity
+Splay.ar(ins, spread, level, center, levelComp: true);
+// or do equal power as float levelComp
+Splay.ar(ins, spread, level, center, levelComp: 0.5); //
+
+// maximum peak safety by equal amplitude comp: level / numchans
+Splay.ar(ins, spread, level, center, levelComp: 1);
+// or calculate by hand:
+Splay.ar(ins, spread, level / ins.size, center, levelComp: false);
+
+// no compensation, just tune level by hand
+Splay.ar(ins, spread, level, center, levelComp: false);
+Splay.ar(ins, spread, level, center, levelComp: 0);
+
+// levelComp by float: level / (numchans ** levelComp),
```

Comment:
suggest: If the `**` operator is unfamiliar, it may be clearer to use `numchans.pow(levelComp)`

Reply:
ok, done